packages/core/root/core-root-server-internal/src/bootstrap.ts

```diff
@@ -79,6 +79,9 @@ export async function bootstrap({ configs, cliArgs, applyConfigOverrides }: Boot
 
   const root = new Root(rawConfigService, env, onRootShutdown);
 
+  const cliLogger = root.logger.get('cli');
+  cliLogger.info('Configurations parsed in this order: ' + env.configs.join(', '));
```

Comment:
nit: since this log will be presented to users, should we explain that the ones on the right override the ones on the left?